latch_cli/services/k8s/utils.py

```diff
@@ -179,35 +196,47 @@ def get_execution_info(execution_id: Optional[str]) -> ExecutionInfoNode:
             """),
             *fragments,
         ),
-        {"createdBy": current_workspace()},
+        {"createdBy": user_config.workspace_id},
     )["runningExecutions"]
 
-    if len(res["nodes"]) == 0:
-        click.secho("You have no executions currently running.", dim=True)
+    nodes = res["nodes"]
+    if nextflow_only:
+        nodes = []
+        for node in res["nodes"]:
+            if node["nfExecutionInfoByExecutionId"] is None:
+                continue
+
+            nodes.append(node)
+
+    if len(nodes) == 0:
+        click.secho(
+            f"You have no {'Nextflow ' if nextflow_only else ''}executions currently"
+            " running.",
+            dim=True,
```

Comment:
we want to also be able to attach to workdir for executions that are not currently running -- should be able to attach to any pvc that is not expired